src/Components/Carousel/Carousel.js

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from "react";
+import { getImageInfo } from "../../Services/ongInfoServices";
+import { Swiper, SwiperSlide } from "swiper/react";
+import SwiperCore, {
+  Autoplay,
+  Navigation,
+  Pagination,
+  Scrollbar,
+  A11y,
+  Parallax,
+} from "swiper";
+import "swiper/swiper-bundle.min.css";
+import "swiper/swiper.min.css";
+import Styles from "./Carousel.module.css";
+SwiperCore.use([Parallax, Autoplay, Navigation, Pagination, Scrollbar, A11y]);
+const Carousel = () => {
+  const [data, setData] = useState([]);
+
+  useEffect(async () => {
+    const dataImage = await getImageInfo();
+    setData(dataImage);
+  }, []);
+
+  console.log(data);
+  return (
+    <div className={Styles.containerCarousel}>
+      <Swiper
+        style={{
+          "--swiper-navigation-color": "#fff",
+          "--swiper-pagination-color": "#fff",
+          textShadow: "0 0 0 #000",
+        }}
+        grid={{
+          gutter: 10,
+          minWidth: 300,
+          maxWidth: 300,
+        }}
+        speed={2000}
+        parallax={true}
+        pagination={{
+          clickable: true,
+        }}
+        loop={true}
+        navigation={true}
+        Autoplay={true}
+        autoplay={{
+          delay: 5000,
+        }}
+      >
```

Comment:
the styles should come from a styles pag

Reply:
changes added